R/add_keep_population_flag.R

```diff
@@ -137,7 +139,7 @@ add_age_group <- function(data, age_var_name) {
   data <- data %>%
     dplyr::mutate(
       age_group = dplyr::case_when(
-        {{ age_var_name }} >= 0 & {{ age_var_name }} <= 4 ~ "0-4",
+        {{ age_var_name }} >= -1 & {{ age_var_name }} <= 4 ~ "0-4",
```

Comment:
Thanks for figuring out this issue, well done!
For age, do we sometimes get people with age -1?

Reply:
I think it's due to age being calculated at the midpoint of the year. So if someone is born in September their age on 30th June is -1. Very fun for us.